content/node/trustlessStaking.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Trustless Third Party Staking
+menuTitle: Trustless Third Party Staking
+weight: 30
+aliases:
+  - /home/node/trustlessStaking
+description: Trustless Third Party Staking utilizes non-custodial feature to stake pokt without handing over custody of tokens to the Node Operator at any stage.
+---
+
+Trustless Third Party Staking utilizes non-custodial feature to stake pokt without handing over custody of tokens to the Node Operator at any stage. With this method, Node Operators have no ability to steal either the stake or the rewards. Clients can also unstake without any intervention of the node operator and get their funds back.
+
+**Before proceeding, please consult with your preffered Node Operator if they support this method of staking.**
+
+This document will show how to stake a node with 15k tokens and later upstake it to 60k.
+We stake with 15k first to minimize any loss resulting from mistake(eg. Typing incorrect output address or public key) 
+
+Non-Custodial Staking method involves the following addresses/wallets
+
+**Output Address**
+Never share the private key of your output address. Make ample backups of this!
+You should already have this, a wallet with your funds. This can also be referred to as the non-custodial address.
+
+
+**Operator Address**
+An Operator Address will be created by the Node Operator.  Node operator will manage(edit Domain/Chains) and run your node using the key of the operator address. 
```

Comment:
I'm wondering if we want to clarify that the node operator will operate and perform maintenance on the node but the non-custodial address can still edit chains, etc. because this reads like they can't. Something like:

Operator Address: Created by the Node Operator, this address is used to manage and run your node. The Node Operator, using the key of the operator address, shares the responsibility for editing the Domain and Chains with the Output Address. However, the Operator Address cannot control the funds; only the Output Address has this capability.